scripts/download-metamath

```diff
@@ -44,7 +44,8 @@ curl -L -o metamath-program.zip \
 # This will be updated, but we need a starting point:
 mkdir -p metamath/
 cd metamath/
-wget -N http://us.metamath.org/mpegif/mmbiblio.html
+#wget -N http://us.metamath.org/mpegif/mmbiblio.html
+wget -N https://raw.githubusercontent.com/metamath/metamath-website-seed/main/mpegif/mmbiblio.html
```

Comment:
wolf@wolf-lammen:~/metamath/xx$ wget -N https://raw.githubusercontent.com/metamath/metamath-website-seed/main/mpegif/mmbiblio.html
--2023-09-05 21:36:41--  https://raw.githubusercontent.com/metamath/metamath-website-seed/main/mpegif/mmbiblio.html
Resolving hostname raw.githubusercontent.com (raw.githubusercontent.com) … 185.199.111.133, 185.199.108.133, 185.199.110.133, ...
Establish connection raw.githubusercontent.com (raw.githubusercontent.com)|185.199.111.133|:443 … connected.
HTTP-Request sent, waiting for reply … 200 OK
Length: 4850 (4,7K) [text/plain]
Saved in ‘mmbiblio.html’.

mmbiblio.html       100%[===================>]   4,74K  --.-KB/s    in 0s      

»Last-modified«-Headline missing -- time stamp turned off.
2023-09-05 21:36:42 (38,2 MB/s) - ‘mmbiblio.html’ saved [4850/4850]

![grafik](https://github.com/metamath/set.mm/assets/30736367/d9f319fd-54de-4d53-8615-6dfc426f6f29)


Seems to be working...